app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionFragment.java

```diff
@@ -239,8 +240,15 @@ private void setupExportToItems(final ViewGroup listHolder) {
     }
 
     private void onImportFromServiceSelected(int serviceId) {
-        if (getParentFragment() == null) return;
-        NavigationHelper.openSubscriptionsImportFragment(getParentFragment().getFragmentManager(), serviceId);
+        FragmentManager fragmentManager;
+
+        if (getParentFragment() == null)
```

Comment:
This code block is repeated three times in the same file, please extract them into the same method.